idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1036,6 +1054,24 @@ struct GetWorkflowExecutionHistoryResponse {
   20: optional binary nextPageToken
 }
 
+struct GetWorkflowExecutionRawHistoryRequest {
+  10: optional string domain
+  20: optional WorkflowExecution execution
+  30: optional binary branchToken
```

Comment:
I don't think these branch tokens are transferable across clusters.  How does the remote cluster gets branchToken for the very first call when requesting history for an execution?  Is the expectation here is to call GetMutableState first before calling GetWorkflowExecutionRawHistory?

Reply:
the purpose of branch token is to guarantee that the returned history will not come from more than one branch, when conflict resolution & history branching is done.

since the feature mentioned above is not implemented, this branchToken will be removed for now.